modules/ngscheckmate/ncm/main.nf

```diff
@@ -0,0 +1,50 @@
+process NGSCHECKMATE_NCM {
+    label 'process_low'
+
+    conda (params.enable_conda ? "bioconda::ngscheckmate=1.0.0" : null)
+    container "${ workflow.containerEngine == 'singularity' && !task.ext.singularity_pull_docker_container ?
+        'quay.io/biocontainers/ngscheckmate:1.0.0--py27r41hdfd78af_3':
```

Comment:
@nf-core/modules-team What is the protocol in this case when the singularity container is taking time to build?